easybuild/main.py

```diff
@@ -182,6 +188,25 @@ def handle_github_options(options, ec_paths):
     return done
 
 
+def _use_color(colorize, stream=sys.stdout):
```

Comment:
this should probably be moved into `tools/options.py`, and be called from the `postprocess` there to determine whether `options.color` should be defined as `True` or `False`, based on the provided (string) value?